packages/headless/src/features/advanced-search-queries/advanced-search-queries-slice.test.ts

```diff
@@ -65,4 +66,42 @@ describe('advanced search parameters', () => {
     );
     expect(nextState).toEqual(expectedQuery);
   });
+
+  describe('#restoreSearchParameters', () => {
```

Comment:
Could add 
```
beforeEach(() => {
  state.aq = 'a';
  state.cq = 'a';
})
```
instead of setting it in all tests

Reply:
Hmm, I would be inclined to place the initialState in a `beforeEach` if there were more than two tests using it, but in this situation, I prefer to have the configuration self-contained instead the unit test.

I also view grouping the `aq` and `cq` to mean "both these params should be set simultaneously", but here this is not the case since the parameters are independent.